vcf/vcf.go

```diff
@@ -139,12 +145,11 @@ func ParseNotes(data string, format []string) []GenomeSample {
 			alleles = strings.SplitN(fields[0], "/", 2)
 			answer[i] = GenomeSample{AlleleOne: common.StringToInt16(alleles[0]), AlleleTwo: common.StringToInt16(alleles[1]), Phased: false, FormatData: fields}
 		} else {
-			//Deal with single haps. There might be a better soltuion, but I think this should work.
 			n, err = strconv.ParseInt(fields[0], 10, 16)
-			if err != nil && n < int64(len(text)) {
+			if err == nil && n < int64(len(text)) {
```

Comment:
I am not sure I understand the comparison to Len(text). I think len(text) is the number of individuals with genotype data in this vcf.  It seems like n should be less than or equal to the number of alleles at this position, which maybe we don't know within this function?